Stefan/src/main/java/JavaBasics.java

```diff
@@ -0,0 +1,83 @@
+public class JavaBasics {
+
+
+    public static void main(String[] args) {
+
+        monthByNumber(6);
+        monthByNumber(12);
+        monthByNumber(44);
+
+        compareStrings("blue", "Blmue");
+
+        calculateCircleArea(5.3);
+
+        displayNumbers(10, 15);
+
+
+    }
+
+    static void monthByNumber(int month) {
```

Comment:
Try to name your methods using verbs. Its widely accepted convention. (ex getMonthByNumber)

Reply:
Done